### ISSUE TITLE
Cannot Connect to RabbitMQ - {utf8,<<"Attach rejected: {address_not_utf8_string,undefined}">>},undefined}

### ISSUE THREAD

**Issue by mgonzaga1990** [quarkusio/quarkus#12358]
**Describe the bug**
This is related to this bug https://github.com/quarkusio/quarkus/issues/7885 but already closed. but not sure if there is a solution or workaround.

I setup my rabbitmq cluster on my kubernetes rpi using the below configs.

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: rabbitmq-config
data:
  enabled_plugins: |
    [rabbitmq_federation,rabbitmq_management,rabbitmq_peer_discovery_k8s,rabbitmq_amqp1_0].
  rabbitmq.conf: |
    loopback_users.guest = false
    listeners.tcp.default = 5672
    amqp1_0.default_user  = guest
    amqp1_0.default_vhost = /
    amqp1_0.protocol_strict_mode = false
    cluster_formation.peer_discovery_backend  = rabbit_peer_discovery_k8s
    cluster_formation.k8s.host = kubernetes.default.svc.cluster.local
    cluster_formation.k8s.address_type = hostname
    cluster_formation.node_cleanup.only_log_warning = true
    ##cluster_formation.peer_discovery_backend = rabbit_peer_discovery_classic_config
    ##cluster_formation.classic_config.nodes.1 = rabbit@rabbitmq-0.rabbitmq.rabbits.svc.cluster.local
    ##cluster_formation.classic_config.nodes.2 = rabbit@rabbitmq-1.rabbitmq.rabbits.svc.cluster.local
    ##cluster_formation.classic_config.nodes.3 = rabbit@rabbitmq-2.rabbitmq.rabbits.svc.cluster.local
---
apiVersion: v1
kind: ServiceAccount
metadata:
  name: rabbitmq
---
kind: Role
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: rabbitmq
rules:
- apiGroups: 
    - ""
  resources: 
    - endpoints
  verbs: 
    - get
    - list
    - watch
---
kind: RoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: rabbitmq
  namespace: rabbits
subjects:
- kind: ServiceAccount
  name: rabbitmq
  namespace: rabbits
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: Role
  name: rabbitmq
---
apiVersion: v1
kind: Secret
metadata:
  name: rabbit-secret
type: Opaque
data:
  # echo -n "cookie-value" | base64
  RABBITMQ_ERLANG_COOKIE: V0lXVkhDRFRDSVVBV0FOTE1RQVc=
---
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: rabbitmq
spec:
  serviceName: rabbitmq
  replicas: 4
  selector:
    matchLabels:
      app: rabbitmq
  template:
    metadata:
      labels:
        app: rabbitmq
    spec:
      serviceAccountName: rabbitmq
      initContainers:
      - name: config
        image: busybox
        command: ['/bin/sh', '-c', 'cp /tmp/config/rabbitmq.conf /config/rabbitmq.conf && ls -l /config/ && cp /tmp/config/enabled_plugins /etc/rabbitmq/enabled_plugins']
        volumeMounts:
        - name: config
          mountPath: /tmp/config/
          readOnly: false
        - name: config-file
          mountPath: /config/
        - name: plugins-file
          mountPath: /etc/rabbitmq/
      containers:
      - name: rabbitmq
        image: rabbitmq:3.8-management
        ports:
        - containerPort: 4369
          name: discovery
        - containerPort: 5672
          name: amqp
        - containerPort: 15672
          name: port-mgmt
        env:
        - name: RABBIT_POD_NAME
          valueFrom:
            fieldRef:
              apiVersion: v1
              fieldPath: metadata.name
        - name: RABBIT_POD_NAMESPACE
          valueFrom:
            fieldRef:
              fieldPath: metadata.namespace
        - name: RABBITMQ_NODENAME
          value: rabbit@$(RABBIT_POD_NAME).rabbitmq.$(RABBIT_POD_NAMESPACE).svc.cluster.local
        - name: RABBITMQ_USE_LONGNAME 
          value: "true"
        - name: RABBITMQ_CONFIG_FILE
          value: "/config/rabbitmq"
        - name: RABBITMQ_ERLANG_COOKIE
          valueFrom:
            secretKeyRef:
              name: rabbit-secret
              key: RABBITMQ_ERLANG_COOKIE
        - name: K8S_HOSTNAME_SUFFIX
          value: .rabbitmq.$(RABBIT_POD_NAMESPACE).svc.cluster.local
        volumeMounts:
        - name: data
          mountPath: /var/lib/rabbitmq
          readOnly: false
        - name: config-file
          mountPath: /config/
        - name: plugins-file
          mountPath: /etc/rabbitmq/
      volumes:
      - name: config-file
        emptyDir: {}
      - name: plugins-file
        emptyDir: {}
      - name: config
        configMap:
          name: rabbitmq-config
          defaultMode: 0755
  volumeClaimTemplates:
  - metadata:
      name: data
    spec:
      accessModes: [ "ReadWriteOnce" ]
      storageClassName: "microk8s-hostpath"
      resources:
        requests:
          storage: 50Mi
---
apiVersion: v1
kind: Service
metadata:
  name: rabbitmq
spec:
  clusterIP: None
  ports:
  - port: 4369
    targetPort: 4369
    name: discovery
  - port: 5672
    targetPort: 5672
    name: amqp
  selector:
    app: rabbitmq
---
apiVersion: v1
kind: Service
metadata:
  name: rabbitmq-svc
  labels:
    app: rabbitmq
spec:
  type: NodePort 
  ports:
  - port: 15672
    name: port-mgmt
    nodePort: 30006
    protocol: TCP
  - port: 5672
    name: port-actual
    nodePort: 30007
    protocol: TCP
  selector:
    app: rabbitmq    
```
**On the configmap** I also tried removing this part
```
    amqp1_0.default_user  = guest
    amqp1_0.default_vhost = /
    amqp1_0.protocol_strict_mode = false
```
reference : https://github.com/rabbitmq/rabbitmq-amqp1.0/blob/v3.7.x/README.md

`application.properties`
```
# Configures the AMQP broker credentials.
amqp-host=192.168.1.150
amqp-port=30007
amqp-username=guest
amqp-password=guest

# Configure the AMQP connector to write to the `prices` address
mp.messaging.outgoing.generated-price.connector=smallrye-amqp
mp.messaging.outgoing.generated-price.address=prices
#mp.messaging.outgoing.generated-price.use-anonymous-sender=false

# Configure the AMQP connector to read from the `prices` queue
mp.messaging.incoming.prices.connector=smallrye-amqp
mp.messaging.incoming.prices.durable=false
```

**Expected behavior**
(Describe the expected behavior clearly and concisely.)

**Actual behavior**
getting the below error
```
2020-09-27 16:13:05.013 [info] <0.580.0> Resetting node maintenance status
2020-09-27 16:20:43.791 [info] <0.1037.0> accepting AMQP connection <0.1037.0> (10.1.7.1:56825 -> 10.1.7.190:5672)
2020-09-27 16:20:43.940 [warning] <0.1043.0> Closing session for connection <0.1037.0>:
{'v1_0.error',{symbol,<<"amqp:invalid-field">>},{utf8,<<"Attach rejected: {address_not_utf8_string,undefined}">>},undefined}
2020-09-27 16:20:46.159 [warning] <0.1037.0> closing AMQP connection <0.1037.0> (10.1.7.1:56825 -> 10.1.7.190:5672):
client unexpectedly closed TCP connection
```

**To Reproduce**
```
mvn io.quarkus:quarkus-maven-plugin:1.8.1.Final:create \
    -DprojectGroupId=org.acme \
    -DprojectArtifactId=amqp-quickstart \
    -Dextensions="amqp"
cd amqp-quickstart
```
then apply this properties
```
# Configures the AMQP broker credentials.
amqp-host=192.168.1.150
amqp-port=30007
amqp-username=guest
amqp-password=guest

# Configure the AMQP connector to write to the `prices` address
mp.messaging.outgoing.generated-price.connector=smallrye-amqp
mp.messaging.outgoing.generated-price.address=prices
#mp.messaging.outgoing.generated-price.use-anonymous-sender=false

# Configure the AMQP connector to read from the `prices` queue
mp.messaging.incoming.prices.connector=smallrye-amqp
mp.messaging.incoming.prices.durable=false
```
**or **

[cannot connect.zip](https://github.com/quarkusio/quarkus/files/5288646/cannot.connect.zip)
(But this is using docker so the host and port is different. But must be the same)

**Additional context**
references followed 
-- https://github.com/rabbitmq/rabbitmq-amqp1.0/blob/v3.7.x/README.md
-- https://github.com/smallrye/smallrye-reactive-messaging/issues/330

**Comment by mgonzaga1990** [comment]
additional log.

```
2020-09-28 00:40:26,642 INFO  [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00208: Deployment done... start processing
2020-09-28 00:40:26,647 INFO  [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00226: Found incoming connectors: [smallrye-amqp]
2020-09-28 00:40:26,648 INFO  [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00227: Found outgoing connectors: [smallrye-amqp]
2020-09-28 00:40:26,649 INFO  [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00229: Channel manager initializing...
2020-09-28 00:40:26,667 INFO  [io.sma.rea.mes.amqp] (Quarkus Main Thread) SRMSG16201: AMQP broker configured to 192.168.1.150:30007 for channel prices
2020-09-28 00:40:26,707 INFO  [io.sma.rea.mes.amqp] (Quarkus Main Thread) SRMSG16201: AMQP broker configured to 192.168.1.150:30007 for channel generated-price
2020-09-28 00:40:26,727 INFO  [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00209: Initializing mediators
2020-09-28 00:40:26,729 DEBUG [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00223: Mediator created for org.acme.amqp.PriceGenerator#generate
2020-09-28 00:40:26,729 DEBUG [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00210: Initializing org.acme.amqp.PriceGenerator#generate
2020-09-28 00:40:26,746 DEBUG [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00213: Registering org.acme.amqp.PriceGenerator#generate as publisher generated-price
2020-09-28 00:40:26,748 DEBUG [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00223: Mediator created for org.acme.amqp.PriceConverter#process
2020-09-28 00:40:26,748 DEBUG [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00210: Initializing org.acme.amqp.PriceConverter#process
2020-09-28 00:40:26,770 INFO  [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00215: Connecting mediators
2020-09-28 00:40:26,771 INFO  [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00216: Attempt to resolve org.acme.amqp.PriceConverter#process
2020-09-28 00:40:26,776 INFO  [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00217: Connecting org.acme.amqp.PriceConverter#process to `[prices]` (org.eclipse.microprofile.reactive.streams.operators.core.PublisherBuilderImpl@549a41e2)
2020-09-28 00:40:26,778 INFO  [io.sma.rea.mes.provider] (Quarkus Main Thread) SRMSG00220: Connecting method org.acme.amqp.PriceGenerator#generate to sink generated-price
2020-09-28 00:40:26,817 INFO  [io.sma.rea.mes.amqp] (Quarkus Main Thread) SRMSG16212: Establishing connection with AMQP broker
2020-09-28 00:40:26,847 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.numHeapArenas: 16
2020-09-28 00:40:26,847 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.numDirectArenas: 16
2020-09-28 00:40:26,847 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.pageSize: 8192
2020-09-28 00:40:26,848 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.maxOrder: 1
2020-09-28 00:40:26,848 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.chunkSize: 16384
2020-09-28 00:40:26,848 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.tinyCacheSize: 512
2020-09-28 00:40:26,848 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.smallCacheSize: 256
2020-09-28 00:40:26,848 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.normalCacheSize: 64
2020-09-28 00:40:26,849 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.maxCachedBufferCapacity: 32768
2020-09-28 00:40:26,849 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.cacheTrimInterval: 8192
2020-09-28 00:40:26,849 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.cacheTrimIntervalMillis: 0
2020-09-28 00:40:26,850 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.useCacheForAllThreads: true
2020-09-28 00:40:26,850 DEBUG [io.net.buf.PooledByteBufAllocator] (vert.x-eventloop-thread-0) -Dio.netty.allocator.maxCachedByteBuffersPerChunk: 1023
2020-09-28 00:40:26,890 DEBUG [io.net.buf.ByteBufUtil] (vert.x-eventloop-thread-0) -Dio.netty.allocator.type: pooled
2020-09-28 00:40:26,890 DEBUG [io.net.buf.ByteBufUtil] (vert.x-eventloop-thread-0) -Dio.netty.threadLocalDirectBufferSize: 0
2020-09-28 00:40:26,891 DEBUG [io.net.buf.ByteBufUtil] (vert.x-eventloop-thread-0) -Dio.netty.maxThreadLocalCharBufferSize: 16384
2020-09-28 00:40:26,893 DEBUG [io.net.uti.NetUtil] (vert.x-eventloop-thread-3) -Djava.net.preferIPv4Stack: false
2020-09-28 00:40:26,893 DEBUG [io.net.uti.NetUtil] (vert.x-eventloop-thread-3) -Djava.net.preferIPv6Addresses: false
2020-09-28 00:40:27,024 DEBUG [io.net.uti.Recycler] (vert.x-eventloop-thread-0) -Dio.netty.recycler.maxCapacityPerThread: 4096
2020-09-28 00:40:27,025 DEBUG [io.net.uti.Recycler] (vert.x-eventloop-thread-0) -Dio.netty.recycler.maxSharedCapacityFactor: 2
2020-09-28 00:40:27,025 DEBUG [io.net.uti.Recycler] (vert.x-eventloop-thread-0) -Dio.netty.recycler.linkCapacity: 16
2020-09-28 00:40:27,025 DEBUG [io.net.uti.Recycler] (vert.x-eventloop-thread-0) -Dio.netty.recycler.ratio: 8
2020-09-28 00:40:27,033 DEBUG [io.net.buf.AbstractByteBuf] (vert.x-eventloop-thread-0) -Dio.netty.buffer.checkAccessible: true
2020-09-28 00:40:27,033 DEBUG [io.net.buf.AbstractByteBuf] (vert.x-eventloop-thread-0) -Dio.netty.buffer.checkBounds: true
2020-09-28 00:40:27,035 DEBUG [io.net.uti.ResourceLeakDetectorFactory] (vert.x-eventloop-thread-0) Loaded default ResourceLeakDetector: io.netty.util.ResourceLeakDetector@70809453
2020-09-28 00:40:27,064 FINE  [org.apa.qpi.pro.eng.imp.SaslImpl] (vert.x-eventloop-thread-0) Handled outcome: SaslImpl [_outcome=PN_SASL_OK, state=PN_SASL_PASS, done=true, role=CLIENT]
2020-09-28 00:40:27,076 FINE  [pro.trace] (vert.x-eventloop-thread-0) IN: CH[0] : Open{ containerId='rabbit@rabbitmq-0.rabbitmq.rabbits.svc.cluster.local', hostname='null', maxFrameSize=32768, channelMax=65535, idleTimeOut=60000, outgoingLocales=null, incomingLocales=null, offeredCapabilities=null, desiredCapabilities=null, properties={cluster_name=rabbit@rabbitmq-0.rabbitmq.rabbits.svc.cluster.local, copyright=Copyright (c) 2007-2020 VMware, Inc. or its affiliates., information=Licensed under the MPL 2.0. Website: https://rabbitmq.com, platform=Erlang/OTP 23.1, product=RabbitMQ, version=3.8.9}}
2020-09-28 00:40:27,079 INFO  [io.sma.rea.mes.amqp] (vert.x-eventloop-thread-0) SRMSG16213: Connection with AMQP broker established
2020-09-28 00:40:27,122 FINE  [pro.trace] (vert.x-eventloop-thread-0) IN: CH[0] : Begin{remoteChannel=0, nextOutgoingId=0, incomingWindow=65535, outgoingWindow=65535, handleMax=65535, offeredCapabilities=null, desiredCapabilities=null, properties=null}
2020-09-28 00:40:27,136 FINE  [pro.trace] (vert.x-eventloop-thread-0) IN: CH[0] : Attach{name='generated-price', handle=0, role=RECEIVER, sndSettleMode=UNSETTLED, rcvSettleMode=FIRST, source=Source{address='null', durable=NONE, expiryPolicy=SESSION_END, timeout=0, dynamic=false, dynamicNodeProperties=null, distributionMode=null, filter=null, defaultOutcome=null, outcomes=[amqp:accepted:list, amqp:rejected:list, amqp:released:list, amqp:modified:list], capabilities=null}, target=Target{address='prices', durable=NONE, expiryPolicy=SESSION_END, timeout=0, dynamic=false, dynamicNodeProperties=null, capabilities=null}, unsettled=null, incompleteUnsettled=false, initialDeliveryCount=null, maxMessageSize=null, offeredCapabilities=null, desiredCapabilities=null, properties=null}
2020-09-28 00:40:27,137 FINE  [pro.trace] (vert.x-eventloop-thread-0) IN: CH[0] : Flow{nextIncomingId=1, incomingWindow=65535, nextOutgoingId=0, outgoingWindow=65535, handle=0, deliveryCount=null, linkCredit=65536, available=null, drain=false, echo=false, properties=null}
2020-09-28 00:40:27,144 DEBUG [io.sma.rea.mes.amqp] (vert.x-eventloop-thread-0) SRMSG16222: Retrieved credits for channel `generated-price`: 65536
2020-09-28 00:40:27,360 DEBUG [io.net.uti.NetUtil] (vert.x-eventloop-thread-3) Loopback interface: lo (Software Loopback Interface 1, 127.0.0.1)
2020-09-28 00:40:27,361 DEBUG [io.net.uti.NetUtil] (vert.x-eventloop-thread-3) Failed to get SOMAXCONN from sysctl and file \proc\sys\net\core\somaxconn. Default: 200
2020-09-28 00:40:27,369 INFO  [io.quarkus] (Quarkus Main Thread) amqp-quickstart 1.0-SNAPSHOT on JVM (powered by Quarkus 1.8.1.Final) started in 3.883s. Listening on: http://0.0.0.0:8080
2020-09-28 00:40:27,370 INFO  [io.quarkus] (Quarkus Main Thread) Profile dev activated. Live Coding activated.
2020-09-28 00:40:27,370 INFO  [io.quarkus] (Quarkus Main Thread) Installed features: [cdi, mutiny, smallrye-context-propagation, smallrye-reactive-messaging, smallrye-reactive-messaging-amqp, vertx]
2020-09-28 00:40:31,802 DEBUG [io.sma.rea.mes.amqp] (RxComputationThreadPool-1) SRMSG16211: Sending AMQP message to address `prices`
2020-09-28 00:40:34,821 SEVERE [io.ver.cor.net.imp.ConnectionBase] (vert.x-eventloop-thread-0) An existing connection was forcibly closed by the remote host: java.io.IOException: An existing connection was forcibly closed by the remote host
	at java.base/sun.nio.ch.SocketDispatcher.read0(Native Method)
	at java.base/sun.nio.ch.SocketDispatcher.read(SocketDispatcher.java:43)
	at java.base/sun.nio.ch.IOUtil.readIntoNativeBuffer(IOUtil.java:276)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:233)
	at java.base/sun.nio.ch.IOUtil.read(IOUtil.java:223)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:358)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:253)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1133)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:350)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:148)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:714)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:576)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)

2020-09-28 00:40:34,824 ERROR [io.sma.rea.mes.amqp] (vert.x-eventloop-thread-0) SRMSG16214: AMQP Connection failure: java.lang.Exception: Connection disconnected
	at io.vertx.amqp.impl.AmqpConnectionImpl.onDisconnect(AmqpConnectionImpl.java:135)
	at io.vertx.amqp.impl.AmqpConnectionImpl.lambda$null$1(AmqpConnectionImpl.java:90)
	at io.vertx.proton.impl.ProtonConnectionImpl.fireDisconnect(ProtonConnectionImpl.java:381)
	at io.vertx.proton.impl.ProtonTransport.handleSocketEnd(ProtonTransport.java:95)
	at io.vertx.core.net.impl.NetSocketImpl.lambda$new$2(NetSocketImpl.java:96)
	at io.vertx.core.streams.impl.InboundBuffer.handleEvent(InboundBuffer.java:237)
	at io.vertx.core.streams.impl.InboundBuffer.write(InboundBuffer.java:127)
	at io.vertx.core.net.impl.NetSocketImpl.handleClosed(NetSocketImpl.java:349)
	at io.vertx.core.net.impl.VertxHandler.lambda$channelInactive$4(VertxHandler.java:162)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:366)
	at io.vertx.core.impl.EventLoopContext.execute(EventLoopContext.java:43)
	at io.vertx.core.impl.ContextImpl.executeFromIO(ContextImpl.java:229)
	at io.vertx.core.impl.ContextImpl.executeFromIO(ContextImpl.java:221)
	at io.vertx.core.net.impl.VertxHandler.channelInactive(VertxHandler.java:162)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:262)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:248)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelInactive(AbstractChannelHandlerContext.java:241)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelInactive(DefaultChannelPipeline.java:1405)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:262)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelInactive(AbstractChannelHandlerContext.java:248)
	at io.netty.channel.DefaultChannelPipeline.fireChannelInactive(DefaultChannelPipeline.java:901)
	at io.netty.channel.AbstractChannel$AbstractUnsafe$8.run(AbstractChannel.java:818)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:497)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by kenfinnigan** [comment]
I might be missing it, but where is the AMQP/RabbitMQ plugin utilized?

**Comment by mgonzaga1990** [comment]
Hi @kenfinnigan I was just trying to connect to my broken when that exception occurs.

**Comment by kenfinnigan** [comment]
Right, but my understanding is that RabbitMQ talks AMQP 0.9, but AMQP in Reactive Messaging talks AMQP 1.0.

The plugin you linked to, I'm not sure whether that's installed in the RabbitMQ broker or somewhere else?

Not sure if @cescoffier has thoughts

**Comment by mgonzaga1990** [comment]
I have raised this also to the rabbitmq team.. and they are lookin into it.. also they are looking on 2 options
https://github.com/rabbitmq/rabbitmq-amqp1.0/issues/102

**Comment by cescoffier** [comment]
Will be fixed with SmallRye Reactive Messaging 2.5.0

**Comment by mgonzaga1990** [comment]
thanks for.the update.

**Comment by cescoffier** [comment]
Was fixed by SmallRye Reactive Messaging 2.5.0 (In Quarkus 1.10)
